docs/sources/review/lint-prose/dictionary.md

```diff
@@ -0,0 +1,66 @@
+---
+description: Learn about the Grafana Labs dictionary template format.
+menuTitle: Dictionary
+review_date: 2024-06-04
+title: Grafana Labs' dictionary
+---
+
+# Grafana Labs' dictionary
+
+The Grafana Labs documentation squad maintains a dictionary used for spell checking.
+The same dictionary is also used to generate some Vale rules from the words metadata.
+
+The dictionary uses the [Hunspell format](https://github.com/hunspell/hunspell) generated from a Jsonnet template.
+
+If a word doesn't yet exist in the dictionary, you can add one by modifying the [dictionary template file](https://github.com/grafana/writers-toolkit/blob/main/vale/dictionary.jsonnet).
+The template file uses the Jsonnet programming language but you don't need to know Jsonnet to add a new word.
+
+## Word metadata
+
+Each word in the dictionary has metadata that describes it.
+The structure of that metadata is a Jsonnet object with the following fields:
+
+| Key                   | Value type | Description                                                                                                                            |
+| --------------------- | ---------- | -------------------------------------------------------------------------------------------------------------------------------------- |
+| `word`                | `string`   | The spelling of the word.                                                                                                              |
+| `affixes`             | `string`   | A concatenation of the letters corresponding to Hunspell affixes. For more information, refer to [Hunspell affixes](#hunspell-affixes) |
+| `po`                  | `string`   | The part of speech descriptor for the word. Known acceptable values include `'noun'`, `'adjective'`, and `'verb'`.                     |
+| `description`         | `string`   | A description of the word.                                                                                                             |
+| `acronym`             | `boolean`  | If `true`, the word is an acronym. Defaults to `false`.                                                                                |
+| `established_acronym` | `boolean`  | If `true`, the acronym doesn't generally need explaining, like the acronym HTTP. Defaults to `false`                                   |
+| `product`             | `boolean`  | If `true`, the word is the name of a product, like Mimir. Defaults to `false`.                                                         |
```

Comment:
Is this for any product (including 3rd party products), or Grafana products?

Reply:
All products at the moment, will clarify that :)